try to improve the "topological models" intro (#242)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

> # Topological Models {#overview}

Reviewer: ```suggestion
# Topological Patterns {#overview}
```

> {{dataflow}} shows an abstract model for communication between an Attester,

Reviewer 2: Is it necessary to change "basic" to "abstract"?
"An abstract model" makes me feel that this model is not practical. But actually, it is a viable model, so I think it's a basic model.

> {{dataflow}} shows an abstract model for communication between an Attester,

Reviewer 3: ```suggestion
{{dataflow}} shows data-flow diagram for communication between an Attester,
```

a Verifier, and a Relying Party. The Attester conveys its Evidence to the Verifier
for appraisal, and the Relying Party gets the Attestation Result from the Verifier.

> This section refines it by describing two reference interaction schemes

Reviewer 3: ```suggestion
This section refines it by describing two reference interaction models
```

> (topological models), as well as one possible compositions thereof. The discussion

Reviewer 3: ```suggestion
(topological models), as well as one example composition thereof. The discussion
```

> This section refines it by describing two reference interaction schemes
> (topological models), as well as one possible compositions thereof. The discussion
> that follows is for illustrative purposes only and does not constrain the
> interactions between RATS roles to the presented patterns.

Reviewer: ```suggestion
This section refines the model by describing two topologies as reference interaction schemes. 
In addition, there is a composition of the models possible.
The discussion that follows is for illustrative purposes only and does not constrain the
interactions between RATS roles to the presented patterns.
```
